tools/testkit/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use fvm_integration_tests::dummy::DummyExterns;
+use fvm_integration_tests::tester::{Account as TAccount, Tester};
+use fvm_ipld_blockstore::MemoryBlockstore;
+use fvm_shared::state::StateTreeVersion;
+use fvm_shared::version::NetworkVersion;
+
+pub mod bundle;
+pub mod fevm;
+
+/// Execution options
+pub struct ExecutionOptions {
+    /// Enables debug logging
+    pub debug: bool,
+    /// Enables gas tracing
+    pub trace: bool,
+    /// Enabls events
+    pub events: bool,
+}
+
+pub type BasicTester = Tester<MemoryBlockstore, DummyExterns>;
+pub struct Account {
+    pub account: TAccount,
+    pub seqno: u64,
+}
+
+pub fn new_tester(bundle_path: String) -> BasicTester {
+    let blockstore = MemoryBlockstore::default();
+    let bundle_cid = match bundle::import_bundle(&blockstore, bundle_path.as_str()) {
+        Ok(cid) => cid,
+        Err(what) => {
+            panic!("error loading bundle: {}", what);
+        }
+    };
+    Tester::new(
+        NetworkVersion::V18,
+        StateTreeVersion::V5,
+        bundle_cid,
+        blockstore,
+    )
+    .unwrap_or_else(|what| {
+        panic!("error creating execution framework: {}", what);
+    })
+}
+
+pub fn create_account(tester: &mut BasicTester) -> Account {
+    let accounts: [TAccount; 1] = tester.create_accounts().unwrap();
```

Comment:
I have a similar approach to creating multiple accounts and just noticed that while the accounts have different actor ID, they had the same Secp256k1 key.

Reply:
Fixed in https://github.com/filecoin-project/ref-fvm/pull/1507/commits/d0203c0d1f70342651c228fd5b658d58f8eab536